tests/com.espressif.idf.ui.test/src/com/espressif/idf/ui/test/operations/EnvSetupOperations.java

```diff
@@ -39,7 +39,7 @@ public static void setupEspressifEnv(SWTWorkbenchBot bot) throws Exception
 		prefrencesShell.bot().tree().getTreeItem("General").getNode("Editors").select();
 		prefrencesShell.bot().tree().getTreeItem("General").getNode("Editors").expand();
 		prefrencesShell.bot().tree().getTreeItem("General").getNode("Editors").getNode("File Associations").select();
-		prefrencesShell.bot().comboBox().setSelection("Text Editor");
+		prefrencesShell.bot().comboBox().setSelection("Plain Text Editor");
```

Comment:
I am not so sure about this. Can't we just open the file in a plain text editor with swt bot without making it default for everything?